.github/workflows/common.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version: ["3.9", "3.10", "3.11"]
-    timeout-minutes: 15
+    timeout-minutes: 20
```

Comment:
Consider adding a comment here explaining why the timeout was increased (e.g., which tests are taking longer). This will help future maintainers understand the context of this change.